common/app/layout/Front.scala

```diff
@@ -254,6 +254,15 @@ case class FaciaContainer(
   }
 
   def hasShowMore = containerLayout.exists(_.hasShowMore)
+
+  /** Nasty hardcoded thing.
+    *
+    * TODO: change Facia Tool to have a dropdown for 'header types', one of which is default, the other CP Scott.
+    *
+    * Then if we end up adding more of these over time, there's an in-built mechanism for doing so. Will also mean apps
+    * can consume this data if they want to.
+    */
+  def showCPScottHeader = dataId == "uk/commentisfree/regular-stories"
```

Comment:
Do we not want to show this on the US or Australian fronts? Also what happens when the name changes?